actors/runtime/src/runtime/fvm.rs

```diff
@@ -358,12 +359,22 @@ where
     }
 
     fn verify_consensus_fault(
-        &self,
+        &mut self,
         h1: &[u8],
         h2: &[u8],
         extra: &[u8],
     ) -> Result<Option<ConsensusFault>, Error> {
-        fvm::crypto::verify_consensus_fault(h1, h2, extra).map_err(|_| Error::msg("no fault"))
+        let ret = fvm::crypto::verify_consensus_fault(h1, h2, extra)
+            .map_err(|_| Error::msg("failed to verify fault"))?;
+        self.charge_gas("verify_consensus_fault_accesses", ret.gas_used);
```

Comment:
We should be charging this gas _inside_ the FVM Kernel, not inside the actor runtime.

Reply:
Makes sense